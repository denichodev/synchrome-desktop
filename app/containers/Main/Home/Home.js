// @flow
import React, { Component } from 'react';

class Home extends Component {
  render() {
    return (
      <div className="animated fadeIn">
        <div className="row">
          <div className="col-md-12">
            <div className="card card-inverse card-primary">
              <div className="card-header">Home</div>
              <div className="card-block">
                Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed
                diam nonummy nibh euismod tincidunt ut laoreet dolore magna
                aliquam erat volutpat. Ut wisi enim ad minim veniam, quis
                nostrud exerci tation ullamcorper suscipit lobortis nisl ut
                aliquip ex ea commodo consequat.
              </div>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

export default Home;
