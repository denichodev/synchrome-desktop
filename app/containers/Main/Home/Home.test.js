import React from 'react';
import Home from './Home';

function setup() {
  const props = {};

  const enzymeWrapper = mount(<Home {...props} />);

  return {
    props,
    enzymeWrapper
  };
}

describe('App', () => {
  it('should match snapshot', () => {
    const { enzymeWrapper } = setup();
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
