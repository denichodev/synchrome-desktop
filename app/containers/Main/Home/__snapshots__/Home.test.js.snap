// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match snapshot 1`] = `
<div
  className="animated fadeIn"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div
        className="card card-inverse card-primary"
      >
        <div
          className="card-header"
        >
          Home
        </div>
        <div
          className="card-block"
        >
          Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat.
        </div>
      </div>
    </div>
  </div>
</div>
`;
